frontend/src/RWD/breakpoints.js

```diff
@@ -0,0 +1,13 @@
+const size = {
+  sm: '480px',
+  md: '768px',
+  lg: '1024px',
+  xl: '1920px',
```

Comment:
Jeszcze jedna drobna uwaga. Czasem niektóre przeglądarki mają problem z takimi ustawieniami. Np. u mnie na monitorze chrome niby ma 1920px,  ale dochodzi jeszcze pasek, ramka przeglądarki, i wtedy tak naprawdę na rozdzielczości 1920px mamy rozdzielczość ustawioną jak na 1024px. I wtedy wygląda to źle.
Dlatego warto dodać wartości trochę mniejsze, uwzględniające te elementy, jak np pasek przewijania

Reply:
![image](https://user-images.githubusercontent.com/41823689/149934452-14184ca7-7fc7-4e38-abf9-86960e557571.png)

Dlatego też użyliśmy mniejszych wartości.

Reply:
Teraz i tablety mają 1024  px, więc zastanowiłbym się raczej nad 1280px